packages/lambdas/src/ihe-gateway-v2-outbound-document-query.ts

```diff
@@ -0,0 +1,80 @@
+import * as Sentry from "@sentry/serverless";
+import { getSecret } from "@aws-lambda-powertools/parameters/secrets";
+import { outboundDocumentQueryReqSchema } from "@metriport/ihe-gateway-sdk";
+import { DQRequestGatewayV2Params } from "@metriport/core/external/carequality/ihe-gateway-v2/ihe-gateway-v2";
+import { createSignSendProcessDQRequests } from "@metriport/core/external/carequality/ihe-gateway-v2/ihe-gateway-v2-logic";
+import { getEnvVarOrFail, getEnvType } from "@metriport/core/util/env-var";
+import { Config } from "@metriport/core/util/config";
+import { capture } from "./shared/capture";
+
+const apiUrl = getEnvVarOrFail("API_URL");
+const documentQueryResponseUrl = `http://${apiUrl}/internal/carequality/document-query/response`;
+
+const privateKeySecretName = Config.getCQOrgPrivateKey();
+const privateKeyPasswordSecretName = Config.getCQOrgPrivateKeyPassword();
+const publicCertSecretName = Config.getCQOrgCertificate();
+const certChainSecretName = Config.getCQOrgCertificateIntermediate();
+
+export const handler = Sentry.AWSLambda.wrapHandler(
+  async ({ patientId, cxId, requestId, dqRequestsGatewayV2 }: DQRequestGatewayV2Params) => {
+    try {
+      console.log(
+        `Running with envType: ${getEnvType()}, requestId: ${requestId}, ` +
+          `numOfGateways: ${dqRequestsGatewayV2.length} cxId: ${cxId} patientId: ${patientId}`
+      );
+
+      const privateKey = await getSecret(privateKeySecretName);
+      const privateKeyPassword = await getSecret(privateKeyPasswordSecretName);
+      const publicCert = await getSecret(publicCertSecretName);
+      const certChain = await getSecret(certChainSecretName);
+      if (
+        !privateKey ||
+        typeof privateKey !== "string" ||
+        !privateKeyPassword ||
+        typeof privateKeyPassword !== "string" ||
+        !publicCert ||
+        typeof publicCert !== "string" ||
+        !certChain ||
+        typeof certChain !== "string"
+      ) {
+        throw new Error("Failed to get secrets or one of the secrets is not a string.");
+      }
+
+      for (const request of dqRequestsGatewayV2) {
```

Comment:
do we even need this schema validation? here and everywhere else?

Reply:
why not - seems like it'd make the code more robust

Reply:
If we use the same type on the lambda and on the caller, I think we don't need the validation